### ISSUE TITLE
Display condition on product page - Switch button - When disabled, it not disable Condition dropdown

### ISSUE THREAD

**Issue by khouloudbelguith** [PrestaShop/PrestaShop#27855]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

Previously reported here: https://github.com/PrestaShop/PrestaShop/pull/27275#issuecomment-1060893404

I have checked the spec: https://github.com/PrestaShop/prestashop-specs/blob/master/content/1.7/back-office/catalog/products/add-edit-v2.md#display-condition-on-product-page

>Display condition on product page - Switch button - When enabled, it enables Condition dropdown, otherwise it is disabled

![image](https://user-images.githubusercontent.com/16067358/157077852-5ef9d407-e273-4106-979b-2174251ee986.png)

It should be 
![image](https://user-images.githubusercontent.com/16067358/157078005-50c22a47-cef0-4484-8d15-f242ac51e081.png)

### Steps to reproduce

1. Go to BO > Catalog > Products V2
2. Add/Edit a product V2
3. In the Specification tab
4. Disable the `Display condition on product page` and check the dropdown is not disabled

### PrestaShop version(s) where the bug happened

develop

### PHP version(s) where the bug happened

7.4

**Comment by jolelievre** [comment]
This feature should be handled with the `DisablingToggler` js component which is implemented in this PR https://github.com/PrestaShop/PrestaShop/pull/27216 So for now it's blocked until the component is merged and available (no need to duplicate it in a new PR, I think we can wait on this one)
